mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -349,7 +349,10 @@ def _serve(model_uri, port, host):
 
 
 def get_cmd(
-    model_uri: str, port: int = None, host: int = None, nworkers: int = None
+    model_uri: str,
+    port: Optional[int] = None,
+    host: Optional[int] = None,
+    nworkers: Optional[int] = None,
```

Comment:
btw do we need this change? It looks like the function signature is not changed at all.

Reply:
+1, Let's put this in a follow-up PR if we want to adjust the type hints.